app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,283 @@
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+// Go to OAuth provider login page
+account.createOAuth2Session('amazon');</code></pre>
+</div>
+
+<p>
+    Notice, the function call for <b>createOAuth2Session</b> is not asynchronous. No need to use <b>await<b>.
+</p>
+
+<p>  
+    If there is already an active anonymous session, the new session will be attached to the logged-in account. If there are no active sessions, the server will attempt to look for an account with the same email address as the email received from the OAuth2 provider and attach the new session to the existing account. If no matching account is found - the server will create a new account.
+</p>
+
+<h3><a href="/docs/authentication#phone" id="phone">Phone</a></h3>
+
+<p>
+    Phone authentication is done using a two-step authentication process. When using phone authentication, the request to authenticate is initiated from the client application, an SMS is sent to the user with a secret key for creating a session. 
+</p>
+
+<p> 
+    Phone authentication requires you to set up an SMS provider to send SMS messages. You will need to configure these <a href="/docs/environment-variables#phone">environment variables</a> and restart your Appwrite containers before you can use phone authentication.
+</p>
+
+<p>
+    The example below shows you how to initiate a phone authentication using the Web SDK.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.createPhoneSession('[USER_ID]', '+16171234567');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    After initiation, the returned username and secret are used to confirm the user. The secret will be a 6 digit number in the SMS message sent to the user.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>const promise = account.updatePhoneSession('[USER_ID]', '[SECRET]');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    If the provided user ID has not been registered, a new account will be created.
```

Comment:
This is incorrect. A new user is not created here
https://github.com/appwrite/appwrite/blob/bf8e113e703084834547ddebe0747108891907f7/app/controllers/api/account.php#L2356